types/three/examples/jsm/webxr/XRHandOculusMeshModel.d.ts

```diff
@@ -0,0 +1,13 @@
+import { Group, Object3D } from './../../../src/Three';
+
+import { HandModelOptions, XRHandModel } from './XRHandModelFactory';
+
+export class XRHandOculusMeshModel {
+    controller: Group;
+    handModel: XRHandModel;
+    bones: Object3D[];
```

Comment:
Looks like these can potentially be `null` (https://github.com/mrdoob/three.js/blob/dev/examples/jsm/webxr/XRHandOculusMeshModel.js#L105).

```suggestion
    bones: Array<Object3D | null>;
```